### DISCUSSION TITLE
make it easier to manage multiple/relationship

### DISCUSSION THREAD

**Discussion by fullyloadedtech** [awcodes/filament-curator#207] [Ideas]
When using the `CuratorPicker` field when you already have 100 images selected, in order to upload a new photo from the same screen you must:
1. click Add media
2. deselect everything first
3. upload the files
4. re-select everything
5. insert/save the full images selection

Please consider adding a way to upload/insert new photos into an already selected multiple relation. Same thing kind of with editing a single image alt/title in the same multiple "Add media" screen - must deselect everything in order to do this. Wish I could make these changes just from one screen instead of clicking each image in the selection and hitting the "Edit" button, which opens every image in a new window (tedious for 100+ images). Also how about select/deselect all buttons?

I am using `CuratorPicker` like this currently:
```
->directory(function (?MyModel $record) {
    if ($record->id) return 'media/mymodel/'. $record->id;
    else return 'media/mymodel';
})
->limitToDirectory(true)
->multiple()
->relationship('photos', 'id')
```

**Comment by awcodes** [comment]
I guess my first question is, how are 100 images getting assigned to a model? I can't imagine that is usable in the field view on the resource.

**Reply by fullyloadedtech** [reply]
It's not very usable and it would be nice if you could toggle to a List View for the `CuratorPicker` like you can for the full Media resource page.

**Comment by Nestoran** [comment]
+1 I think it would be nice to be able to upload new media, without needing to deselect/reselect the currently selected medias.
